Core/src/org/sleuthkit/autopsy/datamodel/GetSCOTask.java

```diff
@@ -42,28 +45,58 @@
         this.listener = listener;
     }
 
+    @Messages({"GetSCOTask.occurrences.defaultDescription=No correlation properties found",
+        "GetSCOTask.occurrences.multipleProperties=Multiple different correlation properties exist for this result"})
     @Override
     public void run() {
         AbstractContentNode<?> contentNode = weakNodeRef.get();
-        
+
         //Check for stale reference
         if (contentNode == null) {
             return;
         }
 
         // get the SCO  column values
         List<Tag> tags = contentNode.getAllTagsFromDatabase();
-        CorrelationAttributeInstance attribute = contentNode.getCorrelationAttributeInstance();
+        CorrelationAttributeInstance fileAttribute = contentNode.getCorrelationAttributeInstance();
 
         SCOData scoData = new SCOData();
         scoData.setScoreAndDescription(contentNode.getScorePropertyAndDescription(tags));
-        scoData.setComment(contentNode.getCommentProperty(tags, attribute));
+        scoData.setComment(contentNode.getCommentProperty(tags, fileAttribute));
         if (!UserPreferences.hideCentralRepoCommentsAndOccurrences()) {
-            scoData.setCountAndDescription(contentNode.getCountPropertyAndDescription(attribute));
+            CorrelationAttributeInstance occurrencesAttribute = null;
+            String description = Bundle.GetSCOTask_occurrences_defaultDescription();
+            if (contentNode instanceof BlackboardArtifactNode) {
+                BlackboardArtifact bbArtifact = ((BlackboardArtifactNode) contentNode).getArtifact();
+                //for specific artifact types we still want to display information for the file instance correlation attribute
+                if (bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_ENCRYPTION_DETECTED.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_ENCRYPTION_SUSPECTED.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_INTERESTING_FILE_HIT.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_METADATA_EXIF.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_KEYWORD_HIT.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_OBJECT_DETECTED.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_EXT_MISMATCH_DETECTED.getTypeID()
+                        || bbArtifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_HASHSET_HIT.getTypeID()) {
+                    occurrencesAttribute = fileAttribute;
```

Comment:
Minor optional comment: It might it be more readable to define a set of these artifact types and then check if the given artifact type is in that set.

Reply:
I personally prefer it this way as opposed to building a set in the constructor (or each time the method is executed). Another option would be a static array and loop through it with explicit break when condition met, but if no one has a strong preference I personally find the medium sized conditional block easier to read since the conditions it is checking are right there.